scipy/stats/tests/test_distributions.py

```diff
@@ -6445,8 +6445,10 @@ def test_only__pdf(self):
 
     def test_only__cdf(self):
         # _pdf is determined from _cdf by taking numerical derivative
-        dummy_distr = _distr2_gen(name='dummy')
-        assert_almost_equal(dummy_distr.pdf(1, a=1), 1)
+        with suppress_warnings() as sup:
+            sup.filter(DeprecationWarning)
```

Comment:
Sorry missed this, can be removed now that we use a private method for `derivative`.